app/serializers/search/classified_listing_author_serializer.rb

```diff
@@ -0,0 +1,7 @@
+module Search
```

Comment:
I put this in its own file because I suspect other records will need Author data with different attributes in the future. I considered making a generic Author serializer and then using [params](https://github.com/Netflix/fast_jsonapi#params), but I quickly saw that getting out of hand the moment you have more than 2 sets of conditionals in the same serializer.